data/themes/darktable.css

```diff
@@ -2525,6 +2520,22 @@ Details :
   color: @thumbnail_fg_color;
 }
 
+/* Set rejected images */
+.dt_thumbnail_rating_6 #thumb_image
+{
+  color: rgba(0,0,0,0.35); /* only the transparency is used to fade the image drawing */
+}
+
+.dt_thumbnail_rating_6 #thumb_ext,
+.dt_thumbnail_rating_6 #thumb_altered,
+.dt_thumbnail_rating_6 .dt_thumb_btn,
+.dt_thumbnail_rating_6#thumb_main:hover #thumb_ext,
+.dt_thumbnail_rating_6#thumb_main:hover #thumb_altered,
+.dt_thumbnail_rating_6#thumb_main:hover .dt_thumb_btn
+{
+  color: rgba(0,0,0,0.2); /* only the transparency is used to fade the image drawing */
```

Comment:
I fear that this feature is only implemented for #thumb_image... So this should have no real effect

Reply:
No, it's not only for #thumb_image as you linked dt_thumbnail_rating_6 id on thumb_main, so all thumbnail and all part, thumb_image and stars, extension name, etc. are inside thumb_main widget.

Reply:
sorry, I see that I've half explained what I have in mind : 
What is not implemented is the transparency thing. iirc only the thumbnail image itself is able to handle the part "only the transparency is used..." as it's some specific cairo code added when drawing the image in the canvas...
For the rest of more "classic" css you can of course use that...

Reply:
For text, it's not necessary to use cairo as CSS is applied directly and naturally on all "text"

Reply:
To precise: it's only here color changes so all items with already color set could be set with that. What you add was a possibility to set that with CSS on image but it's already there for other parts.

Reply:
yes, for text and icons, here, you define the whole color, not only the transparency (just test `rgba(255,0,0,0.2);` to see)
now I wonder if there's not something in css to just add transparency to an existing color... maybe you know that better than me !

Reply:
Of course